core/src/main/scala/org/apache/spark/storage/BlockManagerId.scala

```diff
@@ -37,10 +37,11 @@ import org.apache.spark.util.Utils
 class BlockManagerId private (
     private var executorId_ : String,
     private var host_ : String,
-    private var port_ : Int)
+    private var port_ : Int,
+    private var topologyInfo_ : Option[String])
```

Comment:
we need to document what information this parameter contains (and in what format).